### DISCUSSION TITLE
How to change background color/text color of Actors selected in the World outliner ? ( Darker Mode )

### DISCUSSION THREAD

**Discussion by s-t-a-y** [TheHerobrine/DarkerNodes#149] [Q&A]
![Screenshot (2850)](https://user-images.githubusercontent.com/63308192/118111022-dcaa1380-b402-11eb-8d72-2c8d5b04bfef.png)

**Comment by hugoattal** [comment]
@s-t-a-y This is one of the (very) few changes that require two restart of the engine to be correctly set. This is the primary color that you can set here:
![image](https://user-images.githubusercontent.com/4563971/118116840-1707b680-b3eb-11eb-982f-12e9b51cd94b.png)
or here:
![image](https://user-images.githubusercontent.com/4563971/118116867-2129b500-b3eb-11eb-949e-089a8a77483d.png)

After two restarts, it should looks like this:
![image](https://user-images.githubusercontent.com/4563971/118116915-2edf3a80-b3eb-11eb-81fb-ed91be17ea7b.png)
